Assignment/solution_student/select_fun_part2/js/main.js

```diff
@@ -1,29 +1,57 @@
 // 1.	Slide the header up when it is clicked on.
-
+$('h1').on('click', function () {
+	$('h1').slideUp();
+});
 // 2.	Remove the purplebox class
+$('#box14').removeClass('purplebox');
+
+// Question: Why does $('body') not work in this scenario? That was my first instinct.
 
 // 3.	Hide all of the div’s in #row4
+$('#row4 div').hide();
 
 // 4.	Add a paragraph that says “I have been added” to all boxes with class “add-para” (Hint: you'll need to look up the append() method)
+$('.add-para').append('<p>I have been added</p>');
+
+// This worked with or without the <p> and </p> tag for me. What's the preferred method?
```

Comment:
You're right, that would work fine, but it would place the text 'naked' in that div (http://d.pr/i/1cL6i), so you would just have text with no containing element, which is kinda bad practice, just because _everything_ should have some sort of element around it, and this is absolutely `<p>` material, so I'd prefer to add it as a `<p>` like you did :+1: